src/components/backgroundselector/BackgroundSelectorDirective.js

```diff
@@ -78,6 +74,29 @@ goog.require('ga_topic_service');
             });
             return isValid;
           };
+          var updateDefaultBgOrder = function(bgLayers) {
+            bgLayers = bgLayers ? bgLayers : [];
+            defaultBgOrder = [];
+            bgLayers.forEach(function(bgLayerId) {
```

Comment:
We currently don't use the bglayer information used from the service. Adapting that would require a change in the BOD (re3.topics.background_layers) @gjn your thoughts ?